testing/dockerfiles/bar/Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM    ubuntu
```

Comment:
let's pin a specific `ubuntu:bionic` for instance (in each of these) just for slightly fewer moving parts

Reply:
Sure, doesn't hurt.